src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+
+  const convertedObj = {};
+
+  sourceString.split(';').map((elem) => {
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. It would be better to start each method on a new line for better readability.